orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientSession.java

```diff
@@ -79,4 +112,19 @@ public ClientSession setDocAppSubjectId(Subject docAppSubjectId) {
     public String getClientName() {
         return clientName;
     }
+
+    public String getVtrLocsAsCommaSeparatedString() {
```

Comment:
I don't think this sorts the vtrs - we want to have some kind of consistent ordering, so we always get the same dimensions